From 9337d03750faa25d3ee68aa6ca9982f693e6966c Mon Sep 17 00:00:00 2001
From: nxa07886 <nxa07886@lsv07062.swis.us-cdc01.nxp.com>
Date: Tue, 15 Sep 2020 19:37:49 -0700
Subject: [PATCH] gst-glib-gen.mak: Adapt to backwards incompatible change in
 GNU Make 4.3

GNU Make 4.3 has a backwards incompatible change affecting the use of number signs or hashes (ie., #) inside function invocations.
https://lists.gnu.org/archive/html/info-gnu/2020-01/msg00004.html

Signed-off-by: Neena Busireddy <neenareddy.busireddy@nxp.com>
---
 gst-glib-gen.mak | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/gst-glib-gen.mak b/gst-glib-gen.mak
index 76b2233..bcb4240 100644
--- a/gst-glib-gen.mak
+++ b/gst-glib-gen.mak
@@ -8,7 +8,8 @@
 #glib_gen_decl_banner=GST_EXPORT
 #glib_gen_decl_include=\#include <gst/foo/foo-prelude.h>
 
-enum_headers=$(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+hash:=\#
+enum_headers=$(foreach h,$(glib_enum_headers),\n$(hash)include \"$(h)\")
 
 # these are all the rules generating the relevant files
 $(glib_gen_basename)-marshal.h: $(glib_gen_basename)-marshal.list
-- 
2.17.1

